packages/plugin/src/lib/paystack.ts

```diff
@@ -40,48 +41,76 @@ export default class Paystack {
     this.apiKey = apiKey;
   }
 
+  // protected async requestPaystackAPI<T>(request: Request): Promise<T> {
+  //   const path =
+  //     request.path.replace(/\/$/, "") +
+  //     "/?" +
+  //     new URLSearchParams(request.query).toString();
+
+  //   const options = {
+  //     method: request.method,
+  //     path,
+  //     headers: {
+  //       Authorization: `Bearer ${this.apiKey}`,
+  //       "Content-Type": "application/json",
+  //     },
+  //   };
+
+  //   return new Promise((resolve, reject) => {
+  //     const req = https.request(PAYSTACK_API_PATH, options, res => {
+  //       let data: Uint8Array[] = [];
+
+  //       res.on("data", chunk => {
+  //         data.push(chunk);
+  //       });
+
+  //       res.on("end", () => {
+  //         try {
+  //           resolve(JSON.parse(Buffer.concat(data).toString()) as T);
+  //         } catch (e) {
+  //           reject(e);
+  //         }
+  //       });
+  //     });
+
+  //     req.on("error", e => {
+  //       reject(e);
+  //     });
+
+  //     if (request.body && Object.values(request.body).length > 0) {
+  //       req.write(JSON.stringify(request.body));
+  //     }
+
+  //     req.end();
+  //   });
+  // }
+
   protected async requestPaystackAPI<T>(request: Request): Promise<T> {
     const path =
       request.path.replace(/\/$/, "") +
       "/?" +
       new URLSearchParams(request.query).toString();
+    let data;
 
     const options = {
       method: request.method,
-      path,
+      url: PAYSTACK_API_PATH + path,
       headers: {
         Authorization: `Bearer ${this.apiKey}`,
         "Content-Type": "application/json",
       },
     };
```

Comment:
Since we're using axios now could you remove the request config pre-processing in this method?

I imagine all of this could be replaced with 
```js
const options = {
  method: request.method,
  baseUrl: PAYSTACK_API_PATH
  url: path,
  params: request.query,
  data: request.body
  headers: {
    Authorization: `Bearer ${this.apiKey}`,
    "Content-Type": "application/json",
  },
};
```